libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/dh-processes-detail-item/dh-processes-detail-item.component.html

```diff
@@ -0,0 +1,48 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div class="child-row state-success">
```

Comment:
issue: all processes will have a successful visual state. Will implementing the correct visual state be done in a follow up PR?

Reply:
Yes there is a separate task for it here https://app.zenhub.com/workspaces/batman-60a6105157304f00119be86e/issues/energinet-datahub/geh-metering-point/732